src/commands/env/var/unset.ts

```diff
@@ -52,29 +53,62 @@ export default class ConfigUnset extends Command {
       this.warn(messages.getMessage('flags.environment.deprecation'));
     }
 
-    const appName = await resolveAppNameForEnvironment(targetCompute);
-
     if (argv.length === 0) {
       throw new Errors.CLIError('you must enter a config var key (i.e. mykey)');
     }
 
-    const configPairs = argv.reduce((acc, elem) => {
-      return {
-        ...acc,
-        [elem]: null,
-      };
-    }, {});
+    try {
```

Comment:
Let's wrap ONLY the patch request with the try catch block. We should be able to reduce a lot of duplicated code by doing it this way. Any of the json only stuff (like the json logging) and the non-json stuff (cli-action stuff) can be wrapped by json flag checks at the appropriate phases.